### PR TITLE
Initial commit for a jank indexing procedure for people who don't kno…

### PR DESCRIPTION
**Pull request by jarry-xiao**
…w how to write real code :)

This will primarily be used for demo purposes/one off settings

### CODE REVIEW

File: contracts/sdk/indexer/indexerGummyroll.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  );
+  return subId;
+}
+
+export async function getUpdatedBatch(
+  merkleRoll: OnChainMerkleRoll,
+  db: NFTDatabaseConnection
+) {
+  // If seq > max JS int it's all over :(
+  const seq = merkleRoll.roll.sequenceNumber.toNumber();
```
Comment (ngundotra) [inline contracts/sdk/indexer/indexerGummyroll.ts:45]: use BN?

File: contracts/sdk/indexer/main.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      commitment: "confirmed",
+    });
+
+    console.log(
+        await nftDb.getTreeStmt.all()
+    );
+    await setTimeout(() => {}, 1000);
+  }
+
+  // TODO make sure that we can get proofs from the SQL table
```
Comment (ngundotra) [inline contracts/sdk/indexer/main.ts:106]: main function should `spawn` an indexer & then use `express` to serve proofs

Note: Proof is {root, proof, leafInfo}

**Review by ngundotra — COMMENTED** [review]
😎

File: contracts/sdk/indexer/indexerGummyroll.ts (hunk 1 above)
Comment (jarry-xiao) [reply contracts/sdk/indexer/indexerGummyroll.ts:45]: not hashable

**Pull request merged**